opentracing-api/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "opentracing-api"
+version = "0.1.0"
+authors = ["The OpenTracing Developers"]
+license = "Apache-2.0"
+homepage = "http://opentracing.io"
```

Comment:
Do we want to add the [documentation](https://doc.rust-lang.org/cargo/reference/manifest.html#the-documentation-field-optional) link from the start?
Docs say it is added for us but from what I've seen it was not.

Reply:
Maybe add some `keywords` too?

Reply:
Finally, is a maintain badge to ambitious?
```
# Maintenance: `status` is required Available options are `actively-developed`,
# `passively-maintained`, `as-is`, `none`, `experimental`, `looking-for-maintainer`
# and `deprecated`.
maintenance = { status = "actively-developed" }
```